biggraphite/drivers/cassandra.py

```diff
@@ -1694,12 +1698,19 @@ def touch_metric(self, metric):
                 (metric.name,)
             )
 
-    def map(self, callback, start_key=None, end_key=None, shard=1, nshards=0, errback=None):
+    def map(self, namespaces, start_key=None, end_key=None, shard=1, nshards=0, errback=None):
```

Comment:
This is not a map function anymore. Could you create a stats() function instead that call map() with the right callback. And make the command_stats call stats() only.

Reply:
Done